modules/local/software/cellranger/mkref/main.nf

```diff
@@ -0,0 +1,35 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process CELLRANGER_MKREF {
+
+    label 'process_medium'
+
+    publishDir "${params.outdir}",
+        mode: 'copy',
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    container "streitlab/custom-nf-modules-cellranger:latest"
+
+    input:
+        path(gtf)
+        path(fasta)
+
+    output:
+        path("reference_genome")
+
+    script:
+        mkref_command = "cellranger mkref --genome=reference_genome --genes=${gtf} --fasta=${fasta} --nthreads=${task.cpus}"
+        
```

Comment:
Just as a little tip, you don't need those indents here (after `input:` , `output:` etc.)
So best to remove them for formatting reasons (just having in mind that those should be added to `nf-core/modules` eventually)